examples/providers/DeviceInfoProviderImpl.h

```diff
@@ -46,10 +60,9 @@ class DeviceInfoProviderImpl : public DeviceInfoProvider
         void Release() override { chip::Platform::Delete(this); }
 
     private:
-        EndpointId mEndpoint = 0;
-        size_t mIndex        = 0;
-        char mFixedLabelNameBuf[kMaxLabelNameLength + 1];
-        char mFixedLabelValueBuf[kMaxLabelValueLength + 1];
+        static constexpr size_t kNumSupportedFixedLabels = 0;
+        EndpointId mEndpoint                             = 0;
```

Comment:
Is mEndpointId used anywhere?  Or mIndex?  Don't think they are...

Reply:
I could remove it from here, but will do in the optimization follow-up.

Reply:
Followup fine, yes.